src/index.js

```diff
@@ -390,7 +398,11 @@ class MiniCssExtractPlugin {
     const [chunkGroup] = chunk.groupsIterable;
```

Comment:
Here only the first chunkGroup is used. Maybe this need to be reconsidered. See also comment above.

Reply:
Yeah, i'm trying to look into this. IS there a reason why the order is sorted by `getModuleIndex2(module)`and not `module.id`?

Reply:
My issue get solved if i order after module.id.

Reply:
I will try to make a testcase for it.